src/components/Image.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+const Image = ({ Url, load }) => {
+    return (
+      load ? 
+      <center style={{ marginTop: "40px"}}><h3>Loading</h3></center> :
+            <div className="image" style={{ marginTop: "40px"}}>
+              <img src={Url} alt="" /></div>

```

Comment:
please format your code properly before committing, you can use any JS  formatter extension for your code editor.